build-tools/src/main/java/org/elasticsearch/gradle/plugin/GeneratePluginPropertiesTask.java

```diff
@@ -39,12 +40,13 @@
 
 public abstract class GeneratePluginPropertiesTask extends DefaultTask {
 
-    private static final String PROPERTIES_FILENAME = "plugin-descriptor.properties";
+    public static final String PROPERTIES_FILENAME = "plugin-descriptor.properties";
+    public static final String STABLE_PROPERTIES_FILENAME = "stable-plugin-descriptor.properties";
 
     @Inject
     public GeneratePluginPropertiesTask(ProjectLayout projectLayout) {
-        setDescription("Generate " + PROPERTIES_FILENAME);
-        getOutputFile().convention(projectLayout.getBuildDirectory().file("generated-descriptor/" + PROPERTIES_FILENAME));
+        // TODO I cannot tell if this is stable or not..
```

Comment:
To be honest I would just change it to a generic description like. "Generates Elasticsearch Plugin descriptor files"